src/client_side_reply.cc

```diff
@@ -1644,8 +1644,6 @@ clientReplyContext::buildReplyHeader()
 void
 clientReplyContext::cloneReply()
 {
-    assert(reply == NULL);
```

Comment:
I think this assertion is valid (and, hence, should be restored). Please see the review summary for details.